src/Pos/DB/DB.hs

```diff
@@ -87,20 +92,9 @@ initNodeDBs = do
     prepareGStateBlockExtra initialTip
     prepareLrcDB
     prepareMiscDB
-
--- | Get block corresponding to tip.
-getTipBlock
-    :: (SscHelpersClass ssc, MonadDB m)
-    => m (Block ssc)
-getTipBlock = maybe onFailure pure =<< getBlock =<< getTip
-  where
-    onFailure = throwM $ DBMalformed "there is no block corresponding to tip"
-
--- | Get BlockHeader corresponding to tip.
-getTipBlockHeader
-    :: (SscHelpersClass ssc, MonadDB m)
-    => m (BlockHeader ssc)
-getTipBlockHeader = getBlockHeader <$> getTipBlock
```

Comment:
Well, `Pos.DB` shouldn't import `Pos.Explorer`, actually nothing should import `Pos.Explorer` and it should be moved to `cardano-sl-explorer`. But it's hard to do now, so let's forget about it for now.